packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -254,7 +254,7 @@ export abstract class ProjectRef extends cdk.Construct implements events.IEventR
     /**
      * Allows using build projects as event rule targets.
      */
-    public get eventRuleTarget(): events.EventRuleTargetProps {
+    public asEventRuleTarget(_ruleArn: events.RuleArn, _ruleId: string): events.EventRuleTargetProps {
```

Comment:
Maybe nicer to pass the `Rule` itself, or pass these as props, no?

Reply:
There's a good chance we would want to extract `IEventRuleTarget` into a separate module to prevent cyclic references (which is what I am doing with bucket notifications), so I am trying to standardize on a low-level signature for event targets, and this is the same signature as `IBucketNotificationDestination` will have.